threatmodel.md

```diff
@@ -1,9 +1,16 @@
 ## Threat model
 It is assumed that an attacker may perform one or more the following actions:
 1. intercept and alter network traffic
-2. compromise some set of weak crypto algorithms which are supported in some legacy cases
+2. compromise some set of crypto algorithms, especially known weak algorithms which are supported for legacy purposes
 3. compromise a repository, including gaining access to use any keys stored on the repository
 4. compromise a signing key, for example due to malicious action or accidental disclosure by the key owner
 5. compromise a step in the software supply chain.  This can happen in many different ways, such as by gaining access to the server, compromising the software used in the step of the supply chain, passing different software to a subsequent step than what was intended, or causing an operator to make an error in a step. 
-While it is not always possible to protect against all scenarios, the system should to the extent possible mitigate and/or reduce the damage caused by a successful attack, detect the occurrence of an attack and notify appropriate parties, yet remain usable for parties operating the system.  Furthermore, the system should recover from successful attacks in a way that presents low operational overhead and risk to users.
 
+### Security goals
```

Comment:
I believe these are goals of overall effort. Meaning, we might want to move some of these elements to the scenarios, or a requirements PR. I should not, the usability aspect is critical to our success. 

It would be clean if we can focus the threat model doc as the attack vectors we need to consider.
We can recovering from an attack as a scenario.